### PR TITLE
fix(cb2-7589): potential fix for broken trailers

### PR DESCRIPTION
**Pull request by Ruairidh-BJSS**
## Cant submit new trailer

_One line description_
[link to ticket number](https://dvsa.atlassian.net/browse/CB2-7589)

## Checklist

- [ ] Branch is rebased against the latest develop/common
- [ ] Necessary `id` required prepended with `"test-"` have been checked with automation testers and added
- [ ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Link to the PR added to the repo
- [ ] Delete branch after merge

### CODE REVIEW

File: src/app/features/tech-record/components/tech-record-generate-letter/tech-record-generate-letter.component.ts [hunk 1]
```diff
@@ -45,8 +44,8 @@ export class GenerateLetterComponent implements OnInit {
 
   get reasons(): Array<FormNodeOption<string>> {
     return [
-      { label: 'Authorised', value: Letters.LetterIssueTypeEnum.Authorised },
-      { label: 'Rejected', value: Letters.LetterIssueTypeEnum.Rejected }
+      { label: 'Authorized', value: LettersOfAuth.LetterTypeEnum.Authorization },
```
Comment (andrewf-bjss) [inline src/app/features/tech-record/components/tech-record-generate-letter/tech-record-generate-letter.component.ts:47]: Can we change this back to U.K. spelling please?

```suggestion
      { label: 'Authorised', value: LettersOfAuth.LetterTypeEnum.Authorisation },
```

File: src/app/api/vehicle/model/letters.ts [hunk 2]
```diff
@@ -1,28 +0,0 @@
-/**
```
Comment (naathanbrown) [inline src/app/api/vehicle/model/letters.ts:1]: Aren't these generated? Have you changed the swagger?

**Review by naathanbrown — APPROVED** [review]

**Review by leifkemp-bjss — DISMISSED** [review]

File: src/app/api/vehicle/model/letters.ts (hunk 2 above)
Comment (leifkemp-bjss) [reply src/app/api/vehicle/model/letters.ts:1]: I created the letter components manually, tried basing it off the plates components as closely as possible - including the api models, so if it's not needed then that's fine.

**Review by mcadenas-bjss — DISMISSED** [review]

File: src/app/features/tech-record/components/tech-record-generate-letter/tech-record-generate-letter.component.ts (hunk 1 above)
Comment (Ruairidh-BJSS) [reply src/app/features/tech-record/components/tech-record-generate-letter/tech-record-generate-letter.component.ts]: I've changed the label but the other spelling is generated

**Review by naathanbrown — APPROVED** [review]

**Review by leifkemp-bjss — APPROVED** [review]

**Pull request merged**